PayPal/src/main/java/com/braintreepayments/api/PayPalRequest.java

```diff
@@ -168,69 +125,41 @@ public PayPalRequest shippingAddressEditable(boolean shippingAddressEditable) {
      * <code>zh_TW</code>,
      * <code>zh_XC</code>.
      *
-     * @param localeCode Whether to use a custom locale code.
+     * @param localeCode A locale code to use for the transaction.
      */
-    public PayPalRequest localeCode(String localeCode) {
+    public void setLocaleCode(String localeCode) {
         mLocaleCode = localeCode;
-        return this;
     }
 
     /**
-     * The merchant name displayed in the PayPal flow; defaults to the company name on your Braintree account.
+     * Optional: The merchant name displayed in the PayPal flow; defaults to the company name on your Braintree account.
      *
      * @param displayName The name to be displayed in the PayPal flow.
      */
-    public PayPalRequest displayName(String displayName) {
+    public void setDisplayName(String displayName) {
         mDisplayName = displayName;
-        return this;
     }
 
     /**
-     * Display a custom description to the user for a billing agreement.
+     * Optional: Display a custom description to the user for a billing agreement.
      *
      * @param description The description to display.
      */
-    public PayPalRequest billingAgreementDescription(String description) {
+    public void setBillingAgreementDescription(String description) {
         mBillingAgreementDescription = description;
-        return this;
     }
 
     /**
-     * A custom shipping address to be used for the checkout flow.
+     * Optional: A custom shipping address to be used for the checkout flow.
```

Comment:
We probably shouldn't refer to the checkout flow since this applies to both checkout and vault.

On [iOS](https://github.com/braintree/braintree_ios/blob/2946eebfdef0729dade1f7994ff1869145a04a1d/Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalRequest.h#L76), we say: "Optional: A valid shipping address to be displayed in the transaction flow. An error will occur if this address is not valid."

On [web](https://github.com/braintree/braintree-web/blob/e7c3820d1ba0fe3233bbd1258c6da4fe9502c9d3/src/paypal/external/paypal.js#L155) we say: "Allows you to pass a shipping address you have already collected into the PayPal payment flow."

Maybe we can copy one of those? (It would be even cooler if all three were the same, but... 🙃 )

Reply:
Good call - we'll go with the iOS doc string.